scripts/version

```diff
@@ -9,7 +9,7 @@ if [ -z "$OS" ]; then
     OS="linux"
 fi
 
-FALLBACK_VERSION=v1.20.5+k3s1
+FALLBACK_VERSION=v1.21.13+k3s1
```

Comment:
Will this need changing every release?

Reply:
no, it's only used for dev or drone-pr runs where TAG isn't specified. It looks like it tries to use the GH API to pull the most recent release, but falls back to this version if it can't due to rate limiting.